tests/unittests/Foundation/NSIndexSetTests.mm

```diff
@@ -27,18 +27,20 @@ void verifyIndexSetEqualsArray(NSArray* expected, NSIndexSet* actual) {
     }];
 
     ASSERT_EQ([expected count], callCount);
-    ASSERT_EQ(YES, [actual isEqualToIndexSet:expectedIndexSet]);
+    ASSERT_TRUE([actual isEqualToIndexSet:expectedIndexSet]);
 }
 
 TEST(NSIndexSet, AddIndexesInRange) {
-    NSMutableIndexSet* indexSet = [NSMutableIndexSet new];
+    NSMutableIndexSet* indexSet = [[NSMutableIndexSet new] autorelease];
+    NSMutableIndexSet* expectedIndexSet = [[NSMutableIndexSet new] autorelease];
```

Comment:
Will use this for few tests so we have coverage

---
In reply to: [73382312](https://github.com/Microsoft/WinObjC/pull/796#discussion_r73382312) [](ancestors = 73382312)